packages/wonder-blocks-modal/docs.md

```diff
@@ -1,10 +1,29 @@
 Looking for docs for StandardModal, OneColumnModal, or TwoColumnModal click [here](https://deploy-preview-389--wonder-blocks.netlify.com/#modal)
 
-## Wrapping Modals
+## Accessibility
 
-Often you'll want to define a new modal component by wrapping an existing
-modal component.  These wrapped components will also work with ModalLauncher
-in the same way the default ones do.
+It should follow guidelines from [W3C](https://www.w3.org/TR/wai-aria-practices/#dialog_modal).
+
+### Keyboard Interaction
+When a dialog opens, focus moves to an element inside the dialog. See notes below regarding initial focus placement.
+
+- Tab:
+    - Moves focus to the next tabbable element inside the dialog.
+    - If focus is on the last tabbable element inside the dialog, moves focus to the first tabbable element inside the dialog.
```

Comment:
Which element is the first item?

Reply:
I'm not sure what you mean here. element in terms of element type (e.g. button, input, etc)? or describe which element should be focused according to the current implementation?

Reply:
I guess it doesn't matter a whole lot which is the first item in terms of wrapping around when tabbing.  We can leave this as is for now seeing as have a separate task to improve initial focus behavior.